pkg/controller/services.go

```diff
@@ -237,6 +237,7 @@ func (cont *AciController) staticServiceObjs() apicapi.ApicSlice {
 	bd := apicapi.NewFvBD(cont.config.AciVrfTenant, bdName)
 	bd.SetAttr("arpFlood", "yes")
 	bd.SetAttr("ipLearning", "no")
+	bd.SetAttr("serviceBdRoutingDisable", "yes")
```

Comment:
what happens in the case of older ACI releases where this property is not supported?